--- gtkglext-1.2.0/gdk/gdkglshapes.c~	2004-02-20 09:38:12.000000000 +0000
+++ gtkglext-1.2.0/gdk/gdkglshapes.c	2018-12-02 06:39:51.884573733 +0000
@@ -544,7 +544,7 @@
   {-Z, -X, 0}
 };
 
-static int index[20][3] =
+static int indexX[20][3] =
 {
   {0, 4, 1},
   {0, 9, 4},
@@ -574,7 +574,7 @@
   int i;
 
   for (i = 19; i >= 0; i--) {
-    drawtriangle(i, idata, index, shadeType);
+    drawtriangle(i, idata, indexX, shadeType);
   }
 }
 
